README.md

```diff
@@ -64,7 +82,7 @@ export const RouterHead = component$(() => {
 
 Make sure you remove the `<link rel="icon" type="image/svg+xml" href="/favicon.svg" />` line in your router-head file.
 
-You will need to add `@qwilkdev/pwa/head` to your `tsconfig.json` file in the `compilerOptions.types` option.
+You will need to add `@qwilkdev/pwa` to your `tsconfig.json` file in the `compilerOptions.types` option.
```

Comment:
We should not need this!

Reply:
then how the editor (VSCode/IntelliJ) resolves the types?

Reply:
exports field? it should identify it.

Reply:
the reason why i used to put it here is that the types were not working properly, now they should work! That's why we have `types` in the exports field.

Reply:
done